test/t/test_scp.py

```diff
@@ -81,3 +81,11 @@ def test_live(self, live_pwd, completion):
     @pytest.mark.complete("scp -o Foo=")
     def test_option_arg(self, completion):
         assert not completion  # and no errors either
+
+    @pytest.mark.complete("scp FMWXLlEN8u:", shopt=dict(nullglob=True))
```

Comment:
I guess the intent here is to use a hostname that does not exist in configs to avoid accidentally completing something? Maybe we could use a hostname that would be more explicit about it rather than a random looking string, for example `scp hostname-not-expected-to-exist-in-known-hosts:`?

Reply:
Sorry for the delay, and thank you for the suggestion. I have followed the suggestion.